docs/azurebatch.md

```diff
@@ -17,17 +29,41 @@ The Azure Batch account name.
 
 The Azure Batch account key.
 
-### `--storage_name`
+### `-w`
 
-The Azure Blob Storage name.
+The Azure Blob container to be used as Nextflow work directory (`-w az://work`).
 
-### `--sas_token`
+### `--vm_type`
 
-The Azure Blob Storage shared access signature token.
+VM size to use with Nextflow autopool or when creating a worker pool in Azure Batch. Make sure your Azure account has sufficient quota. Defaults to `Standard_D8s_v3`.
```

Comment:
Add link [Sizes for virtual machines in Azure](https://docs.microsoft.com/en-us/azure/virtual-machines/sizes)